.github/resources/crd_rbac.yaml

```diff
@@ -13,7 +13,7 @@ rules:
       - releases
       - releaseplans
       - releaseplanadmissions
-      - releasestrategies
```

Comment:
This was just a leftover of when we completely removed release strategies, right? I wonder why it wasn't an issue. Does the `kind` cluster not care that you mention non-existing resources when applying a Role? I guess so. Maybe it's normal since you might apply roles before applying crds?

Reply:
Yes, that is my understanding

Reply:
ok, cool